packages/deployer/package.json

```diff
@@ -29,7 +29,7 @@
     "dotenv": "^10.0.0",
     "ethers": "^5.4.5",
     "lodash": "^4.17.21",
-    "ora": "^6.0.1",
+    "ora": "^5.4.1",
```

Comment:
Why the downgrade here? I updated this recently with dependabot https://github.com/api3dao/airnode/pull/594/files

Reply:
@amarthadan mentioned that it broke the deployer package (but this could have been in a seprate PR to avoid confusion).

Reply:
Ah sorry about that. I wasn't aware

Reply:
@andreogle I mentioned it in the commit message :slightly_smiling_face: `ora` is now supporting only EMS modules and our current TS configuration doesn't transpile to that. So, I figured it's better to downgrade the not that important dependency instead of changing our TS configuration :smile: